src/hotspot/share/compiler/abstractCompiler.hpp

```diff
@@ -165,6 +165,18 @@ class AbstractCompiler : public CHeapObj<mtCompiler> {
     ShouldNotReachHere();
   }
 
+  // Notifies this compiler that the current thread (`thread`) is about to stop.
+  // The current thread currently holds the CompileThread_lock.
+  virtual void stopping_compiler_thread(CompilerThread* thread) {
```

Comment:
If `thread` is the current thread then I'd recommend calling it `current` so that this is clear.

Reply:
Fixed: https://github.com/openjdk/jdk/pull/8262/commits/f448c2f6f859aea772a748458cbd336a8744afed